web/permissions/permissions.go

```diff
@@ -299,6 +299,29 @@ func listPermissions(c echo.Context) error {
 	return json.NewEncoder(resp).Encode(doc)
 }
 
+func showPermissions(c echo.Context) error {
+	inst := middlewares.GetInstance(c)
+
+	current, err := middlewares.GetPermission(c)
+	if err != nil {
+		return err
+	}
+
+	doc, err := permission.GetByID(inst, c.Param("permdocid"))
+	if err != nil {
+		return err
+	}
+
+	if doc.ID() != current.ID() && doc.SourceID != current.SourceID {
```

Comment:
So we want to forbid fetching a permission doc when using a token associated with a wildcard permission?